src/jit/codegenlegacy.cpp

```diff
@@ -15952,7 +15968,7 @@ void        CodeGen::genEmitHelperCall(unsigned    helper,
     void * addr = NULL, **pAddr = NULL;
 
     // Don't ask VM if it hasn't requested ELT hooks 
-#if defined(_TARGET_ARM_) && defined(DEBUG)
+#if defined(_TARGET_ARM_) && defined(DEBUG) && defined(PROFILING_SUPPORTED)
```

Comment:
The compProfilerMethHnd member is only defined if PROFILING_SUPPORTED is defined.